content/en/docs/concepts/containers/_index.md

```diff
@@ -18,7 +17,9 @@ run it.
 Containers decouple applications from underlying host infrastructure.
 This makes deployment easier in different cloud or OS environments.
 
-
+Each {{< glossary_tooltip text="node" term_id="node" >}} in a Kubernetes
+cluster runs the containers that form part of the
+[Pods](/docs/concepts/workloads/pods/) assigned to that node.
```

Comment:
non-blocking nit: add clarification that containers of a Pod run on the same node
```suggestion
Each {{< glossary_tooltip text="node" term_id="node" >}} in a Kubernetes
cluster runs the containers that form the
[Pods](/docs/concepts/workloads/pods/) assigned to that node.
Containers in a Pod are co-located and co-scheduled to run on the same node.
```